Examples/apps/SegmentUIKitExample/SegmentUIKitExample/Tab1ViewController.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ViewController.swift
+//  SegmentUIKitExample
+//
+//  Created by Brandon Sneed on 4/8/21.
+//
+
+import UIKit
+
+class Tab1ViewController: UITableViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
+    @IBAction func queryAction(_ sender: Any) {
+        let alertController = QueryAlertController(title: "I know you like pie", message: "Everyone likes pie", preferredStyle: .alert)
```

Comment:
What is this for?

Reply:
it's an example of customized screen tracking.  just added a comment saying such.